batch/batch/utils.py

```diff
@@ -87,6 +82,26 @@ def cost_from_msec_mcpu(msec_mcpu):
     return (msec_mcpu * 0.001 * 0.001) * (total_cost_per_core_hour / 3600)
 
 
+def worker_memory_cpu_cost_per_core_hour(worker_type, preemptible):
+    if preemptible:
+        if worker_type == 'standard':
+            cost_per_core_hour = 0.01
+        elif worker_type == 'highcpu':
+            cost_per_core_hour = 0.0075
+        else:
+            assert worker_type == 'highmem'
+            cost_per_core_hour = 0.0125
+    else:
+        if worker_type == 'standard':
+            cost_per_core_hour = 0.04749975
+        elif worker_type == 'highcpu':
+            cost_per_core_hour = 0.0354243
+        else:
+            assert worker_type == 'highmem'
+            cost_per_core_hour = 0.0591515
+    return cost_per_core_hour
```

Comment:
For functions like this, I find it less confusing when we don't use elif and else and just directly return when we find the right condition. That avoids the unnecessary variable and keeps all the conditions aligned (nb: `if ` is three characters but `elif ` and `else ` are 5).